lgc/include/lgc/state/ResourceUsage.h

```diff
@@ -56,6 +56,12 @@ enum class BasicType : unsigned {
   Uint8,       // 8-bit unsigned integer
 };
 
+// Represents the type of location
+enum class LocationType : unsigned {
```

Comment:
// Represents the type of input/output packing
LocationType -> PackType

Reply:
This is used in PipelineState.h, so should be defined there. But my other comment suggests removing it entirely anyway.

Reply:
Removed.